workshops/beat_maker/README.md

```diff
@@ -0,0 +1,418 @@
+---
+name: 'Beat Maker'
+description: 'Make a web drumpad/beat maker in HTML/CSS & jQuery'
+author: '@emmanuel39hanks'
+---
+
+# Creating a DrumPad/Beat Maker in HTML, CSS & jQuery
+
+![Beat Maker Preview](https://i.ibb.co/r0JLpWB/Annotation-2020-09-20-200719.png)
+
+_^ Preview of the DrumPad/Beat Maker we are going to be creating_
+
+In this workshop, you’re going to create a drumpad/beat maker that plays actual sounds, in less than 230 lines of code.
+
+[Link to demo](https://AssuredStudiousNotifications--five-nine.repl.co)
+
+## Getting started
+
+We’re going to be using HTML, CSS & jQuery(a JavaScript library designed to simplify HTML DOM tree traversal and manipulation, as well as event handling, CSS animation, and Ajax).
```

Comment:
This would be a great spot to link to jQuery’s site. I also think folks won’t know what those techniques are—maybe using a phrase like “changing elements on a webpage” or something?